.travis/deploy_minikube.sh

```diff
@@ -14,15 +14,35 @@ set -x
 # NOTE: This script was originally copied from the Cojaeger-operator build
 # https://github.com/jaegertracing/jaeger-operator/blob/master/.travis/setupMinikube.sh
 
-# socat is needed for port forwarding
-sudo apt-get update && sudo apt-get install socat && sudo apt-get install conntrack
-
 export MINIKUBE_VERSION=v1.8.2
 export KUBERNETES_VERSION=v1.17.3
 
-sudo mount --make-rshared /
-sudo mount --make-rshared /proc
-sudo mount --make-rshared /sys
+source /etc/os-release
+
+if [ "${ID}" == "ubuntu" ]
+then 
+    # socat is needed for port forwarding
+    sudo apt-get update && sudo apt-get install socat && sudo apt-get install conntrack
+    sudo mount --make-rshared /
+    sudo mount --make-rshared /proc
+    sudo mount --make-rshared /sys
+elif [ "${ID}" == "centos" ]
+then
+    dnf install -y sudo
+    sudo dnf -y update && sudo dnf -y install socat conntrack 
+    dnf install -y dnf-plugins-core
+    sudo dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    sudo dnf install -y docker-ce docker-ce-cli containerd.io --nobest
```

Comment:
why docker-ce and not podman and/or buildah?

Reply:
originally it was on deploying Ubuntu (as it for travis runs).
We can change to podman when running on Centos

Reply:
By the look of it, minikube support for podman is still WIP.
We can see 2 things:
1. Using the podman driver is experimental
2. even though the driver is podman, the run-time is docker and changing it to cri-o still looks for docker (https://github.com/kubernetes/minikube/issues/6795).

We are using MINIKUBE_VERSION=v1.8.2 and KUBERNETES_VERSION=v1.17.3 due to https://github.com/kubernetes/minikube/issues/7828 

running with podman results in:
```
+ main@./1.sh:70 cat /root/.minikube/config/config.json
{
    "WantNoneDriverWarning": false,
    "WantUpdateNotification": false,
    "container-runtime": "cri-o",
    "driver": "podman",
    "vm-driver": "none"
}+ main@./1.sh:72 minikube version
minikube version: v1.8.2
commit: eb13446e786c9ef70cb0a9f85a633194e62396a1
+ main@./1.sh:74 minikube start --kubernetes-version=v1.17.3
😄  minikube v1.8.2 on Centos 8.1.1911
    ▪ MINIKUBE_VERSION=v1.8.2
✨  Using the podman (experimental) driver based on user configuration
E0705 15:30:53.797867   10377 cache.go:106] Error downloading kic artifacts:  error loading image: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```
once podman support in minikube will be better, and we will be able to upgrade the minikube version we can switch to podman on Centos8.

Reply:
I have opened a new issue to track this: https://github.com/noobaa/noobaa-core/issues/6075

Reply:
I have seen similar problems with minikube+podman, it does not really work well yet.